src/sqlancer/questdb/test/QuestDBQueryPartitioningWhereTester.java

```diff
@@ -18,6 +18,11 @@ public QuestDBQueryPartitioningWhereTester(QuestDBGlobalState state) {
     @Override
     public void check() throws SQLException {
         super.check();
+        if (select == null) {
```

Comment:
Here, I don't understand how it can happen that `select` is `null` and how it relates to the comment. Could you please explain?